src/group_theory/perm/sign.lean

```diff
@@ -259,6 +261,11 @@ begin
     exact hmul_swap _ _ _ hxy.1 (ih _ ⟨rfl, λ v hv, hl.2 _ (list.mem_cons_of_mem _ hv)⟩ h1 hmul_swap) }
 end
 
+/-- Like `swap_induction_on`, but with the composition on the right of `f` -/
```

Comment:
Can you copy the docstring for `swap_induction_on` here to make this self-contained? And maybe the docstring for `swap_induction_on` should mention this version as well.

Reply:
I figured there was little point to this given that `swap_induction` appears immediately above in both the file and in anyone's auto-complete list.

Reply:
That's true. I was thinking of someone hovering over `swap_induction_on'`.

Reply:
Those people could always jump to definition by holding ctrl and click while they hover :). I'll let you make the final call - do you want me to copy the docstring anyway?

Reply:
Yes, please.